containers/server-postgresql-image/Dockerfile

```
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: MIT
+#!BuildTag: uyuni/server-postgresql:latest
+
+ARG BASE=registry.opensuse.org/opensuse/postgres:16
+# or ARG BASE=registry.suse.com/suse/postgres:16
+FROM $BASE
```

I'm not sure this is going to work as expected.

from the IBS prjconf we define:
`BuildFlags: dockerarg:BASE=bci/bci-base:15.7` 
while on OBS we have: 
`BuildFlags: dockerarg:BASE=registry.suse.com/bci/bci-base:15.6` 
and to my knowledge, this value in the Dockerfile is going to be replaced with what comes from the prjconf. 

We could use something like this I  think:
```suggestion
ARG BASEDB=registry.opensuse.org/opensuse/postgres:16
FROM $BASEDB
```

and in the prjconf  we could define the 2 different images for IBS and OBS

IBS:
`BuildFlags: dockerarg:BASEDB=registry.suse.com/suse/postgres:16`

OBS:
`BuildFlags: dockerarg:BASEDB=registry.opensuse.org/opensuse/postgres:16`

---

I agree and have a local change to use `PG_BASE` as a name for that image base